libs/patchwheel/parse_test.go

```diff
@@ -265,18 +265,26 @@ func TestParseWheelFilename(t *testing.T) {
 		},
 	}
 
+	prefixes := []string{
+		"",
+		"./",
+		"hello/world/",
+	}
+
 	for _, tt := range tests {
-		t.Run(tt.filename, func(t *testing.T) {
-			info, err := ParseWheelFilename(tt.filename)
-			if tt.wantErr {
-				require.Error(t, err)
-			} else {
-				require.NoError(t, err)
-				require.Equal(t, tt.wantDistribution, info.Distribution, "distribution mismatch")
-				require.Equal(t, tt.wantVersion, info.Version, "version mismatch")
-				require.Equal(t, tt.wantTags, info.Tags, "tags mismatch")
-			}
-		})
+		for _, prefix := range prefixes {
```

Comment:
Double nesting is not really needed here. Can you add one additional case to the table?

Reply:
fair point, https://github.com/databricks/cli/pull/2521/commits/972530fe67a34d2ac35674da7404888751c58a6f